### DISCUSSION TITLE
Unable to access former assignments through Classroom GUI since July 1

### DISCUSSION THREAD

**Discussion by fuhrmanator** [community/community#130889] [GitHub Education]
### Hi everyone,

Since July 1, my team of TAs and I are unable to access already started assignments in Classrooms. Whenever we click on an assignment in a classroom, there's an error:

> PATCH https://api.github.com/repos/ORGANIZATION/CLASSROOM: 403 - Resource not accessible by integration // See: https://docs.github.com/rest/repos/repos#update-a-repository

Here's a screen capture:

![image](https://github.com/community/community/assets/7606540/138655a8-a8f0-4a60-98ae-025cbd7dbca0)

We can get to the repos (a github search with the prefix for the assignment will find them easily), but there's no dashboard to see status of commits, etc. I also noticed this is a problem for all of my past courses (last summer, etc.) which had starter code in a different organization. I use a different organization for starter code because TAs have Owner roles in the classroom organization, which means a rogue TA could delete the starter code repo.

I'm pretty sure this PATCH error is related to the updates requiring starter code be in the same organization that rolled out on July 1. However, I didn't think that it would affect assignments that are already started. My university has trimesters, and we began our summer projects in May, knowing that July 1 changes were coming, but assuming that it would not affect projects that were _already_ started. 

Currently, my TAs (and I) are confused as to how to proceed. Is there a way to resolve this on https://docs.github.com/rest/repos/repos#update-a-repository or is that a misleading error message?

Thanks for any help.

**Comment by alexisseurin** [comment]
Your GitHub App token does not have the necessary permissions to access the requested resource. You need to verify GitHub App permissions and check repository permissions.

**Reply by fuhrmanator** [reply]
> You need to verify GitHub App permissions and check repository permissions.

Sure, that's what the error literally means. It says:

> The fine-grained token must have the following permission set:
> * "Administration" repository permissions (write)

But usually GitHub classrooms doesn't require _us_ to create tokens. This problem happened to me and all TAs in my classroom (we're 4 people). It has happened to several other instructors at my university in other classrooms that began in May. 

My _guess_ is that because the starter code was/is in another organization, the permissions (running when I click the assignment) aren't sufficient to access it?
